pennylane/beta/devices/default_tensor.py

```diff
@@ -99,8 +99,8 @@ class DefaultTensor(Device):
     # pylint: disable=attribute-defined-outside-init
     name = "PennyLane TensorNetwork simulator plugin"
     short_name = "default.tensor"
-    pennylane_requires = "0.17"
-    version = "0.17.0"
+    pennylane_requires = "0.16"
+    version = "0.16.0"
```

Comment:
```suggestion
    pennylane_requires = "0.17"
    version = "0.17.0"
```

Should we consider having the devices simply use `_version` rather than having it hardcoded manually?